spec/unit/graph/simple_graph_spec.rb

```diff
@@ -528,31 +528,31 @@ def empty_graph(graph)
     end
 
     def one_vertex_graph(graph)
-      graph.add_vertex(:a)
```

Comment:
I didn't find any cases where Symbol preservation is needed. If it is, it should be done using rich_data semantics instead of relying on YAML. What if we serialize with JSON instead for instance?